Sources/String+Sweetness.swift

```diff
@@ -1,4 +1,5 @@
 public extension String {
+    @available(*, deprecated: 9, message: "Please use the native count method directly instead.")
```

Comment:
Without passing any kind of information about the version of *what* `9` is, what does this wind up defaulting to? Swift 9? Xcode 9? iOS 9?

Reply:
Gone were the days of good documentation and reliability. Apple documentation just says that `deprecated` takes a platform or language version version, but doesn't specify how you differentiate them. I tried setting it to `swift 4.0`, but it doesn't work. So I just used 9 (assuming iOS 9), since that is the target version for this framework. 😞 If you know how I can specify that it's swift 4, I'd appreciate it.

Reply:
Maybe you need to do `@available(swift *, deprecated: 4...` - [`swift` is definitely a valid availability modifier](https://developer.apple.com/library/content/documentation/Swift/Conceptual/Swift_Programming_Language/Attributes.html#//apple_ref/doc/uid/TP40014097-CH35-ID347)

Reply:
That's what their docs say, could not find a way to make it work.

Reply:
Ok, testing here, it defaulted to language version. And it worked if I did it like this:

`@available(swift, deprecated: 4.0, message: "Please use the native count method directly instead.")`